velox/common/memory/MmapAllocator.cpp

```diff
@@ -345,6 +345,7 @@ bool MmapAllocator::allocateContiguousImpl(
         (useMmapArena_ ? "true" : "false"),
         folly::errnoStr(errno));
     VELOX_MEM_LOG(ERROR) << errorMsg;
+    VELOX_MEM_LOG(ERROR) << toString();
```

Comment:
Can you add MmapAllocator info into errorMsg? Thanks!

Reply:
Done!